src/ophys_etl/workflows/pipeline_modules/motion_correction.py

```diff
@@ -20,6 +21,16 @@
 class MotionCorrectionModule(PipelineModule):
     """Wrapper around motion correction module"""
 
+    def __init__(self,
```

Comment:
It's not needed since the subclass constructor is just passing to the superclass constructor